apps-client/mobile/components/poll/poll-card.tsx

```diff
@@ -0,0 +1,353 @@
+import React from 'react'
+import { View } from 'react-native'
+
+import { Badge } from '@pple-today/ui/badge'
+import { Button } from '@pple-today/ui/button'
+import { Icon } from '@pple-today/ui/icon'
+import { cn } from '@pple-today/ui/lib/utils'
+import { Text } from '@pple-today/ui/text'
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { useRouter } from 'expo-router'
+import { ChevronRightIcon, Clock3Icon, MessageCircleQuestionIcon } from 'lucide-react-native'
+
+import type { FeedItemPoll } from '@api/backoffice/app'
+import { exhaustiveGuard } from '@app/libs/exhaustive-guard'
+
+import { PollOptionItem, PollOptionResult } from './poll-option'
+
+type PollItem = FeedItemPoll['poll']
+
+dayjs.extend(duration)
+
+export const PollCardContent = (props: { feedItem: FeedItemPoll }) => {
+  const router = useRouter()
+  const [isEnded, setIsEnded] = React.useState(dayjs().isAfter(dayjs(props.feedItem.poll.endAt)))
+
+  const renderSeeMoreOptions = () => {
+    if (props.feedItem.poll.options.length > 3) {
+      return (
+        <Button
```

Comment:
This button is too customized lets use `Pressable` no?
```suggestion
        <Pressable
```